README.md

```diff
@@ -3,6 +3,52 @@
 The members' data API is a Play app that manages and retrieves supporter attributes associated with a user.  
 It runs on https://members-data-api.theguardian.com/.
 
+
+## How do we handle Zuora 40 concurrent requests limit?
+
+When users visit dotcom a check is made for entitlements. For example, should ads be displayed. This results in high
+load on members-data-api which we need to manage. In particular Zuora does not provide a caching mechanism whilst
+at the same time having 40 concurrent requests limit. Note this limit applies globally across our systems, and Zuora
+does not provide segregation by a particular client. Hence, the onus is on us to manage the limit.
+
+There are few ways we try to manage the load 
+1. dotcom cookies that expire after 24 hours
+1. mem-data-api DynamoDB table with particular TTL
+1. mechanism to control number of concurrent Zuora requests each mem-data-api instance can make (currently there are 6)
+
+With current load management we hit Zuora around 1000 per minute.
+
+### Dotcom cookies caching
+
+TODO: Tom to explain. Maybe list the names of cookies with example of their content
```

Comment:
will do

Reply:
done in https://github.com/guardian/members-data-api/commit/de517f2d81ffd582f598b35e78ce7f7dd835b4b6